docs/ru/operations/table_engines/odbc.md

```diff
@@ -0,0 +1,29 @@
+# ODBC
```

Comment:
why not symlink?

Reply:
Because we had a problems with commenting when I replaced symlink with the file.